beacon-chain/forkchoice/doubly-linked-tree/forkchoice_test.go

```diff
@@ -742,3 +742,21 @@ func TestForkchoice_UpdateJustifiedBalances(t *testing.T) {
 	require.DeepEqual(t, balances, f.justifiedBalances)
 
 }
+
+func TestForkChoice_UnrealizedJustifiedPayloadBlockHash(t *testing.T) {
+	ctx := context.Background()
+	f := setup(0, 0)
+
+	st, blkRoot, err := prepareForkchoiceState(ctx, 0, [32]byte{'a'}, params.BeaconConfig().ZeroHash, [32]byte{'A'}, 1, 1)
+	require.NoError(t, err)
+	require.NoError(t, f.InsertNode(ctx, st, blkRoot))
+
+	f.store.unrealizedJustifiedCheckpoint.Root = [32]byte{'a'}
+	got, err := f.UnrealizedJustifiedPayloadBlockHash()
+	require.NoError(t, err)
+	require.DeepEqual(t, [32]byte{'A'}, got)
```

Comment:
Question here, isn't `Equal` enough for slices?

Reply:
It is. Updated to `Equal`